griptape/labs/tools/aws_kendra/tool.py

```diff
@@ -0,0 +1,45 @@
+import json
+import logging
+from griptape.artifacts import BaseArtifact, TextArtifact, ErrorArtifact
+from griptape.core import BaseTool
+from griptape.core.decorators import activity
+from schema import Schema, Literal
+from attr import define, field
+import boto3
+
+@define
+class AwsKendraClient(BaseTool):
+    index_id: str = field(default=None, kw_only=True)
```

Comment:
Will it work with `None`? If so the type should be `Optional[str]`. If not, then no default?

Reply:
`index_is` is required, updating.